content/manuals/security/for-admins/single-sign-on/manage.md

```diff
@@ -54,24 +54,39 @@ aliases:
 
 ## Manage users
 
-{{< tabs >}}
-{{< tab name="Admin Console" >}}
+> [!IMPORTANT]
+>
+> SSO has Just-In-Time (JIT) Provisioning enabled by default unless you have [disabled it](/security/for-admins/provisioning/just-in-time/#sso-authentication-with-jit-provisioning-disabled). This means your users are auto-provisioned to your organization.
+>
+> You can change this on a per-app basis. To prevent auto-provisioning users, you can create a security group in your IdP and configure the SSO app to authenticate and authorize only those users that are in the security group. Follow the instructions provided by your IdP:
+>
+> - [Okta](https://help.okta.com/en-us/Content/Topics/Security/policies/configure-app-signon-policies.htm)
+> - [Entra ID (formerly Azure AD)](https://learn.microsoft.com/en-us/azure/active-directory/develop/howto-restrict-your-app-to-a-set-of-users)
+>
+> Alternatively, see [Manage how users are provisioned](/manuals/security/for-admins/single-sign-on/manage.md).
```

Comment:
This link loops back to the top of the topic instead of a section. I wasn't able to find a section called "manage how users are provisioned", so it's a little ambiguous. Is it supposed to link to [Manage provisioning](https://deploy-preview-21274--docsdocker.netlify.app/security/for-admins/single-sign-on/manage/#manage-provisioning)?

Reply:
good catch! I changed it to link to the provisioning overview guide